awx/main/tasks.py

```diff
@@ -1394,12 +1394,12 @@ def status_handler(self, status_data, runner_config):
             with disable_activity_stream():
                 self.instance = self.update_model(self.instance.pk, job_args=json.dumps(runner_config.command), job_cwd=runner_config.cwd, job_env=job_env)
         elif status_data['status'] == 'error':
-            result_traceback = status_data.get('result_traceback', None)
-            if result_traceback:
-                from awx.main.signals import disable_activity_stream  # Circular import
-
-                with disable_activity_stream():
```

Comment:
Do we want to capture this kind of automated update in the activity stream?

Reply:
the activity stream won't capture entries that modify fields with editable=False, which was the case for both of these.

Really, all unified job fields should be non-editable, but the _original_ implementation of the API was that you did a POST to create the job record, and then you did a POST to the `/api/v2/jobs/N/start/` endpoint. Because of that, you could still edit the job fields while status=new, meaning that every field related to the user values or the job template were editable. We don't allow this anymore, but the fields are what they are. Anyway, neither `result_traceback` or `job_explanation` would have fit that prior definition of editability, because the user wouldn't be able to set them.

I tested it and it did seem to not produce an activity stream entry. But you know, a unit test would be good for this.